frontend/packages/noobaa-storage-plugin/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@console/noobaa-storage-plugin",
+  "version": "0.0.0-fixed",
+  "description": "Noobaa Storage - TDB",
```

Comment:
Let's not leave TBD in there... I'd just use "Noobaa Storage" for now

Reply:
NooBaa Storage - Object data service for Kubernetes hybrid and multi-cloud environments

Reply:
Update the description as per the suggestion.